test/e2e/operator_test.go

```diff
@@ -0,0 +1,279 @@
+package e2e
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	framework "github.com/openshift/cluster-csi-snapshot-controller-operator/test/framework"
+	"github.com/openshift/cluster-csi-snapshot-controller-operator/test/helpers"
+	kapierrs "k8s.io/apimachinery/pkg/api/errors"
+
+	"k8s.io/kubernetes/test/e2e/storage/drivers"
+
+	volumesnapshotsv1beta1 "github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
+)
+
+const (
+	snapshotGroup      = "snapshot.storage.k8s.io"
+	snapshotAPIVersion = "snapshot.storage.k8s.io/v1beta1"
+
+	operatorNamespace = "openshift-csi-snapshot-controller-operator"
+	operatorName      = "csi-snapshot-controller-operator"
+)
+
+var (
+	retryInterval        = time.Second * 5
+	timeout              = time.Second * 120
+	cleanupRetryInterval = time.Second * 1
+	cleanupTimeout       = time.Second * 5
+	poll                 = 2 * time.Second
+	size                 = 1000000
+)
+
+func TestCSISnapshotControllerOperator(t *testing.T) {
+	ctx := framework.NewClientSet("")
```

Comment:
`ctx` is usually used for `context.Context`, I'd used another name here

Reply:
I've adjusted this to be `client` throughout the test.